src/updates/titanium/sdk.ts

```diff
@@ -89,10 +89,26 @@ export async function checkInstalledVersion (validateSelected = false): Promise<
 }
 
 export async function checkLatestVersion (): Promise<SDKInfo> {
-	const { stdout } = await runTiCommand([ 'sdk', 'list', '--releases', '--output', 'json' ]);
-	const releases = Object.keys(JSON.parse(stdout).releases).map(name => name.replace('.GA', ''));
-	const latest = releases.sort(semver.rcompare)[0];
+	let releases;
+	try {
+		const { stdout } = await runTiCommand([ 'sdk', 'list', '--releases', '--output', 'json' ]);
```

Comment:
I probably should have only returned GA releases by default and added a flag to add beta/rc releases.